src/component/strategies/StrategiesList/StrategiesList.tsx

```diff
@@ -265,81 +270,105 @@ export const StrategiesList = () => {
-                    }
-                    permission={UPDATE_STRATEGY}
-                    tooltipProps={{ title: 'Edit strategy' }}
-                >
-                    <Edit />
-                </PermissionIconButton>
-            }
-            elseShow={
-                <Tooltip title="You cannot edit a built-in strategy" arrow>
-                    <div>
-                        <IconButton disabled size="large">
-                            <Edit titleAccess="Edit strategy" />
-                        </IconButton>
-                    </div>
-                </Tooltip>
-            }
-        />
-    );
+                }
+                elseShow={
+                    <Tooltip title="You cannot edit a built-in strategy" arrow>
+                        <div id={id} role="tooltip">
+                            <IconButton
+                                disabled
+                                size="large"
+                                aria-describedby={id}
+                            >
+                                <Edit titleAccess="Edit strategy" />
+                            </IconButton>
+                        </div>
+                    </Tooltip>
+                }
+            />
+        );
+    };
 
-    const deleteButton = (strategy: IStrategy) => (
-        <ConditionallyRender
-            condition={strategy?.editable}
-            show={
-                <PermissionIconButton
-                    onClick={() => onDeleteStrategy(strategy)}
-                    permission={DELETE_STRATEGY}
-                    tooltipProps={{ title: 'Delete strategy' }}
-                >
-                    <Delete />
-                </PermissionIconButton>
-            }
-            elseShow={
-                <Tooltip title="You cannot delete a built-in strategy" arrow>
-                    <div>
-                        <IconButton disabled size="large">
-                            <Delete titleAccess="Delete strategy" />
-                        </IconButton>
-                    </div>
-                </Tooltip>
-            }
-        />
-    );
+    const deleteButton = (strategy: IStrategy) => {
+        const id = createStrategyId(strategy, 'delete');
+        return (
+            <ConditionallyRender
+                condition={strategy?.editable}
+                show={
+                    <PermissionIconButton
+                        onClick={() => onDeleteStrategy(strategy)}
+                        permission={DELETE_STRATEGY}
+                        tooltipProps={{ title: 'Delete strategy' }}
+                    >
+                        <Delete />
+                    </PermissionIconButton>
+                }
+                elseShow={
+                    <Tooltip
+                        title="You cannot delete a built-in strategy"
+                        arrow
+                    >
+                        <div id={id} role="tooltip">
```

Comment:
Hmm, I might be understanding [MDN's guide to the tooltip role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tooltip_Role#associated_wai-aria_roles_states_and_properties), but it says:

> The element that serves as the tooltip container has role="tooltip" set.

Shouldn't the `role=tooltip` go on the `Tooltip` component, then? (full disclosure: I haven't checked the Tooltip component code).

MDN's example shows something like this, at least:

```html
<label for="password">Password:</label>
<input aria-describedby="passwordrules" id="password" type="password" />
<div role="tooltip" id="passwordrules">
  <p>
      Password Rules:
  </p>
  <ul>
    <li> Minimum of 8 characters</li>
    <li> Include at least one lowercase letter, one uppercase letter, one number and one special character</li>
    <li>Unique to this website</li>
  </ul>
</div>
```

Reply:
Ah you're right, the role is for the container. :+1: